src/main/java/bio/terra/cli/context/WorkspaceContext.java

```diff
@@ -100,29 +101,29 @@ public void deleteWorkspace() {
   }
 
   /**
-   * Getter for the Terra workspace display name. Returns empty string if the display name is not
+   * Getter for the Terra workspace display name. Returns empty Optional if the display name is not
    * defined.
    *
    * @return the Terra workspace display name
    */
   @JsonIgnore
-  public String getWorkspaceDisplayName() {
-    return terraWorkspaceModel == null || terraWorkspaceModel.getDisplayName() == null
-        ? ""
-        : terraWorkspaceModel.getDisplayName();
+  public Optional<String> getWorkspaceDisplayName() {
+    return terraWorkspaceModel == null
+        ? Optional.empty()
+        : Optional.ofNullable(terraWorkspaceModel.getDisplayName());
```

Comment:
nit: I think slightly more idiomatic would be `return Optional.ofNullable(terraWorkspaceModel).map(WorkspaceDescription::getDisplayName);` Note that [Optional#map](https://docs.oracle.com/javase/8/docs/api/java/util/Optional.html#map-java.util.function.Function-) returns an empty optional if the mapping function result is null.

ditto on description